### ISSUE TITLE
Resolve official linkset resources (headers/data)

### ISSUE THREAD

**Issue by shaneseaton** [CSIRO-enviro-informatics/loci-cache-scripts#35]
The header.ttl files for each linkset currently lives in two places. One in in the linkset creator scripts, and one in the official linkset repository. 

Having them in two spots is clearly bad. Need one to reference the other. 

Bigger question is what do we do about the linksets repositories? Are they dead? Should we be updating them when we rerun the linkset creators? Should they simple point to the S3 bucket?

**Comment by jyucsiro** [comment]
Is this now resolved @shaneseaton ?

**Comment by shaneseaton** [comment]
I don't believe the issue of having a single point of truth for the header information is resolved.

eg. The Addr2Catch linkset has the official repo, in which the header is:
https://github.com/CSIRO-enviro-informatics/addrcatch-linkset/blob/master/header.ttl

But the linkset generator splits the header in two and uses a template system to generate the header information actually used in the datacache:
https://github.com/CSIRO-enviro-informatics/loci-cache-scripts/blob/master/docker/linksets/gnaf2geofab/resources/header_imports.ttl.template
https://github.com/CSIRO-enviro-informatics/loci-cache-scripts/blob/master/docker/linksets/gnaf2geofab/resources/header_statements.ttl.template

The version of the header in the linkset-repo, and indeed all the data in the linkset-repo, should be auto generated as an output of the linkset creation process. It shouldn't be manually created independently of the linkset generator code.

Also, these question should be addressed...
> Bigger question is what do we do about the linksets repositories? Are they dead? Should we be updating them when we rerun the linkset creators? Should they simple point to the S3 bucket?
